src/client/pydaos/SConscript

```diff
@@ -4,7 +4,7 @@ import daos_build
 def build_shim_module(env, lib_prefix, version):
     """Build PyDAOS shim module for the specified python version"""
 
-    if GetOption('help'):
+    if GetOption('help') or GetOption('clean'):
```

Comment:
This doesn't work.   You need to only wrap the ParseConfig stuff.   You still want it to execute the other stuff or it won't clean anything